apps/liveontwitch/live_on_twitch.star

```diff
@@ -277,7 +284,10 @@ def get_followed_streams(refresh_token, user_id):
         },
     )
     if res.status_code == 200:
-        return res.json()["data"]
+        streams = res.json()["data"]
+
+        # Filter out streams in the ignore list
+        return [stream for stream in streams if stream["user_name"] not in ignore_list]
```

Comment:
Does `ignore_list` ever get converted from the comma-separated string to a list? I suppose it probably doesn't matter because `not in` would still work with a substring. My one other small suggestion would be to use `.lower()` on both the `ignore_list` and `stream["user_name"]` to make the check case-insensitive. Otherwise, I think this looks good.

Reply:
Agreed. I've updated the pr with those changes.